lib/features/announcement/ui/announcements_list/setting_page.dart

```diff
@@ -14,32 +15,271 @@ class _VocelSetting extends State<VocelSetting>{
   Widget build(BuildContext context) {
     // TODO: implement build
     return Scaffold(
+      appBar: AppBar(
+// backgroundColor: Colors.blue, // AppBar Color
+        automaticallyImplyLeading: false,
+        leading: SizedBox(
+          height: 60,
+          width: 60,
+          child: FloatingActionButton(
+            backgroundColor: Colors.transparent, // Back Button Color
+            elevation: 0,
+            child: const Icon(
+              Icons.arrow_back_rounded,
+              color: Colors.white, // Back Icon Color
+              size: 30,
+            ),
+            onPressed: () {
+              Navigator.pop(context);
+            },
+          ),
+        ),
+        title: const Text(
+          "Settings",
```

Comment:
pull from resource file to keep internationalized

Reply:
done